release_bot/init_repo.py

```diff
@@ -86,14 +86,15 @@ def create_conf(self):
         gitchangelog = input(
             'would you like to use gitchangelog to generate next-gen changelogs? (Y/n):'
             )
-        if gitchangelog.lower() == 'y' or gitchangelog == '':
-            self.conf['gitchangelog'] = True
-        else:
-            self.conf['gitchangelog'] = False
+        self.conf['gitchangelog'] = bool(
+            gitchangelog.lower() == 'y' or gitchangelog == ''
+            )
 
         cwd = os.getcwd()
         self.release_conf['author_email'] = run_command_get_output(cwd, f'git config user.email')[1]
+        self.release_conf['author_email'] = self.release_conf['author_email']
```

Comment:
I think you forgot to pull the changes after you accepted my suggestions (which created new commit).
And therefore you returned this previously removed line.

don't worry, it happened to me too recently :-)